apps/src/applab/designProperties.jsx

```diff
@@ -52,9 +55,10 @@ export default class DesignProperties extends React.Component {
     const isScreen = (elementType === elementLibrary.ElementType.SCREEN);
     // For now, limit duplication to just non-screen elements
 
+    const onlyOneScreen = (elementUtils.getScreens().length === 1);
```

Comment:
I think it'd be better to check `this.props.screenIds.length` here, rather than using `getScreens()` which actually queries the DOM.

Reply:
Done.